src/utils/helpers.ts

```diff
@@ -0,0 +1,5 @@
+import { Todo } from '../types/Todo';
+
+export const filterByStatus = (todos: Todo[], status: boolean) => {
```

Comment:
Is it obvious why `status` is `boolean`?

Reply:
renamed it to `isCompleted`